src/containers/Dashboard/index.js

```diff
@@ -1,11 +1,11 @@
-import React from 'react';
 import PropTypes from 'prop-types';
-
+import React from 'react';
 import DeviceStatsCard from './components/DeviceStatsCard';
+import LineChart from './components/lineChart';
 import PieChart from './components/PieChart';
 import styles from './index.module.scss';
 
-const Dashboard = ({ titleList, statsArr, pieChartData }) => {
+const Dashboard = ({ pieChartTitle, statsArr, pieChartData, lineChartData, lineChartTitle }) => {
```

Comment:
Just put the titles inside the data so you don't need all these extra arrays for titles